src/controllers/userController.js

```diff
@@ -2,7 +2,7 @@
 import localStorage from 'localStorage';
 import uuid from 'uuid/v4';
 import bcrypt from 'bcryptjs';
-import sendVerificationEmail from 'utils/email.helper';
+import { sendVerificationEmail, sendNotificationEmail } from 'utils/email.helper';
```

Comment:
```suggestion
import { sendVerificationEmail, sendNotificationEmail } from 'utils/emailHelper';
```
Would you please use camelCase

Reply:
Thanks, @hakoemmy I resolved it.